internal/wasm/interpreter/interpreter.go

```diff
@@ -549,76 +550,31 @@ func (me *moduleEngine) Call(m *wasm.ModuleContext, f *wasm.FunctionInstance, pa
 		}
 	}()
 
-	for _, param := range params {
-		ce.push(param)
-	}
 	if f.Kind == wasm.FunctionKindWasm {
+		for _, param := range params {
```

Comment:
I chose to not attempt to normalize calling wasm as it Is less similar between JIT and interpreter. However, someone can try later and maybe it isn't worthless to attempt.